src/sourcekit-lsp/LanguageClientManager.ts

```diff
@@ -247,7 +247,7 @@ export class LanguageClientManager {
         const sourcekit: langclient.Executable = {
             command: serverPath,
             args: lspConfig.serverArguments.concat(sdkArguments),
-            options: { env: { ...process.env, ...swiftRuntimeEnv() } },
+            options: { env: { ...configuration.swiftEnvironmentVariables, ...swiftRuntimeEnv() } },
```

Comment:
`process.env` should also be passed here

Reply:
Just found we need to do the same with `execSwift`. It looks like we completely clear all other environment variables by setting one with `env` flags.

Reply:
Fixed